triggers/createFile.js

```diff
@@ -0,0 +1,252 @@
+		}
+	} catch (error) {
+		z.console.log("Error fetching events: " + error.message);
+		throw error;
+	}
+
+	const testWebHook = {
+		url: `${process.env.BASE_URL}/service/platform/notification/v1.0/webhook-configs/test`,
+		method: "POST",
+		body: {
+			url: "https://www.example.com",
+			secret: "",
+		},
+	};
+
+	try {
+		let testHookResponse = await z.request(testWebHook);
+		if (testHookResponse.status === 200) {
+			try {
+				const webhookConfigResponse = await z.request({
+					url: `${process.env.BASE_URL}/service/platform/notification/v1.0/webhook-configs`,
+					method: "POST",
+					body: {
+						events: [...eventIds],
+						isActive: true,
+						name: uuidv4(),
+						secret: "",
+						url: bundle.targetUrl,
+					},
+				});
+
+				if (webhookConfigResponse.status === 200) {
+					hookID = webhookConfigResponse.data.webhookConfigId;
+					return webhookConfigResponse.data;
+				} else {
+					throw new Error(
+						`Failed to create webhook configuration. Status: ${webhookConfigResponse.status}`
+					);
+				}
+			} catch (error) {
+				z.console.log("Error creating webhook configuration: " + error.message);
+				throw error;
+			}
+		}
+	} catch (error) {
+		z.console.log("Error creating test WEBHOOK: " + error.message);
+		throw new Error(
+			`Failed to create a test webhook configuration. Status: ${error}`
+		);
+	}
+};
+
+const performList = async (z, bundle) => {
+	const { PixelbinConfig, PixelbinClient } = require("@pixelbin/admin");
+	const zapier = require("zapier-platform-core");
+	zapier.tools.env.inject();
+
+	body = {
+		event: {
+			name: "file",
+			type: "create",
+			traceId: "8f2937c8-92f7-47c3-a8eb-71c50408fa3d",
+		},
+		payload: {
+			orgId: 7671,
+			type: "file",
+			name: "pb_result.png",
+			path: "",
+			fileId: "pb_result.png",
+			access: "public-read",
+			tags: [],
+			metadata: {
+				source: "direct",
+			},
+			format: "png",
+			assetType: "image",
+			size: 538309,
+			width: 2660,
+			height: 1360,
+			context: {
+				steps: [],
+				req: {
+					headers: {},
+					query: {},
+				},
+				meta: {
+					format: "png",
+					size: 538309,
+					width: 2660,
+					height: 1360,
+					space: "srgb",
+					channels: 4,
+					depth: "uchar",
+					density: 144,
+					isProgressive: false,
+					resolutionUnit: "inch",
+					hasProfile: true,
+					hasAlpha: true,
+					extension: "png",
+					contentType: "image/png",
+					assetType: "image",
+					isImageAsset: true,
+					isAudioAsset: false,
+					isVideoAsset: false,
+					isRawAsset: false,
+					isTransformationSupported: true,
+				},
+			},
+			isOriginal: true,
+		},
+		public_id: `${process.env.CDN_URL}/v2/polished-hat-8f9bd4/original/dummy_image.png`,
+	};
+
+	let defaultPixelBinClient = new PixelbinClient(
+		new PixelbinConfig({
+			domain: `${process.env.BASE_URL}`,
+			apiSecret: bundle.authData.apiKey,
+		})
+	);
+	let temp = await defaultPixelBinClient.assets.listFilesPaginator({
```

Comment:
get only 1 file, this will return top 10 files

Reply:
resolved